applications/dashboard/models/class.tagmodel.php

```diff
@@ -530,6 +532,7 @@ public function getDiscussions($tag, $limit, $offset) {
             ->join('Tag t', 't.TagID = td.TagID')
             ->whereIn('t.Name', $tags)
             ->limit($limit, $offset)
+            ->orderBy($sortField, $sortDirection)
```

Comment:
What about using these parameters when getting discussions, a couple lines down? If I were to pass in "DateInserted" for `$sortField`, I would expect it to be applied to the discussion, but as it stands, it'll be applied to when the discussion was tagged. I don't know if `DiscussionModel::get` would be flexible enough to allow you to set sorting options. You might need to switch it up to `DiscussionModel::getWhere`.

Reply:
Now that we're using the sort for the discussions, we probably don't need to sort the TagDiscussion results too.